src/main/java/com/welovecoding/netbeans/plugin/editorconfig/editor/csl/EditorConfigPropertyStructureItem.java

```diff
@@ -0,0 +1,86 @@
+package com.welovecoding.netbeans.plugin.editorconfig.editor.csl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import javax.swing.ImageIcon;
+import org.netbeans.modules.csl.api.ElementHandle;
+import org.netbeans.modules.csl.api.ElementKind;
+import org.netbeans.modules.csl.api.HtmlFormatter;
+import org.netbeans.modules.csl.api.Modifier;
+import org.netbeans.modules.csl.api.StructureItem;
+
+/**
+ *
+ * @author junichi11
+ */
+public class EditorConfigPropertyStructureItem implements StructureItem {
+
+  private final String key;
+  private final String value;
+  private final int startPosition;
+  private final int endPosition;
+
+  public EditorConfigPropertyStructureItem(String key, String value, int startPosition, int endPosition) {
```

Comment:
JavaDoc on `key`, `value`, `startPosition` and `endPosition` would be nice. We should also consider whether or not it makes sense to pass an object instead of 4 parameters. The [Clean Code book](http://www.amazon.co.uk/dp/0132350882) recommends to have a maximum number of 2 parameters. If there are more then the parameter should become an object / class.